qemu/tests/cfg/test_SMM_enabled.cfg

```diff
@@ -0,0 +1,8 @@
+- test_SMM_enabled:
+    type = test_SMM_enabled
+    no Host_RHEL.6 Host_RHEL.7 Host_RHEL.8.u0 Host_RHEL.m8.u1 Host_RHEL.m8.u2 Host_RHEL.m8.u3 Host_RHEL.m8.u4 Host_RHEL.m8.u5 Host_RHEL.m8.u6
+    vms = ""
+    auto_cpu_policy = "virttest"
```

Comment:
It's for VM, it seems not to need this time.

Reply:
> It's for VM, it seems not to need this time.

This policy will execute before booting a vm, so we can keep it to avoid libvirt issue to breaking this.